scripts/legislation/africa/eritrea_english.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^CHAPTER I/,$p'
+}
+
+function add_square_brackets_to_list_numbers {
+    sed -E 's/(\()([0-9]+)(\))/[\2]/g' 
+}
+
+function append_pipe_to_article_title {
+  sed -E 's/Article: 37/Article 37/' | \
+  sed -E 's/^(Article .*)$/\1|/'
```

Comment:
The second line should be further indented because it is a continuation of the first line (due to the backslash).